src/containers/App/index.js

```diff
@@ -27,11 +28,27 @@ import { requestCategories } from './actions';
 export const AppContainer = ({ requestCategoriesAction }) => {
   // on each component render, see if the current session is authenticated
   authenticate();
+  const history = useHistory();
 
   useEffect(() => {
     requestCategoriesAction();
   }, []);
 
+  useEffect(() => {
+    const unlisten = history.listen(() => {
+      defer(() => {
```

Comment:
Why would you defer the `scrollTo`?

Reply:
Also, did you test this with both `PUSH` and `POP` actions? It might not be necessary on `POP`, because browsers usually restore scroll position on that action.

Reply:
that is because of the use  of defer in the wizard form component: 
that is needed for correct rendering of visible and not visible questions

it will bite you in the ass

Reply:
How exactly will it bite me in the ass?

Reply:
alleen de wizard form heeft de defer nodig.  vreemd. wat zal ik doen?

Reply:
I guess you need to find out why navigating the wizard form requires us to defer the scrolling of the page.